internal/impl/pure/input_read_until.go

```diff
@@ -104,9 +123,15 @@ func newReadUntilInput(conf input.Config, mgr bundle.NewManagement, log log.Modu
 			return nil, fmt.Errorf("failed to parse check query: %w", err)
 		}
 	}
+	var idleTimeout time.Duration = -1
+	if len(conf.ReadUntil.IdleTimeout) > 0 {
+		if idleTimeout, err = time.ParseDuration(conf.ReadUntil.IdleTimeout); err != nil {
+			return nil, fmt.Errorf("failed to parse idle_timeout string: %v", err)
+		}
+	}
 
-	if check == nil {
-		return nil, errors.New("a check query is required")
+	if check == nil && idleTimeout < 0 {
```

Comment:
Wouldn't it be better to have `<= 0` and initialise `idleTimeout` to `0` instead of `-1`? Not sure if `0` duration makes sense here.

Reply:
That also works. At first, my goal was to set it as negative to disable it, but I later realized that I didn't need to do that because we can use the empty string as a default value, so I guess I just forgot to change it to 0 since it still works like this. I'll update it, thanks